src/Service/Verify.php

```diff
@@ -50,18 +49,13 @@ public function fileWithSignature(SplFileInfo $file, SplFileInfo $signature) : b
             file_get_contents($signature->getPathname()),
         );
 
-        switch (true) {
-            case false === $result:
-            case $result[0]['summary'] !== 0:
-                throw new \RuntimeException(sprintf(
-                    'Verification between "%s" and "%s" failed!',
-                    $signature->getFilename(),
-                    $file->getFilename()
-                ));
-                break;
-            default:
+        if ($result === false || $result[0]['summary'] !== 0) {
+            throw new RuntimeException(sprintf(
+                'Verification between "%s" and "%s" failed!',
+                $signature->getFilename(),
+                $file->getFilename()
```

Comment:
Not sure if the error message is the best we can do.
"Verification between" doesn't sound like correct english to me..?

And: Own exception ;)

Reply:
Changed